src/etc/inc/filter.inc

```diff
@@ -483,6 +483,23 @@ function filter_configure_sync($delete_states_if_needed = true) {
 		$_grbg = exec("/sbin/pfctl -t bogonsv6 -T kill 2>/dev/null");
 	}
 
+	// get aliases names from config
+	$aliasesnames = array();
+	foreach ($config['aliases']['alias'] as $alias) {
+		array_push($aliasesnames, $alias['name']);
+	}
+	// get aliases names from pfctl -sT
+	$pftables = explode("\n",shell_exec("/sbin/pfctl -sT"));
```

Comment:
Use `exec("/sbin/pfctl -sT", $pftables);` instead of this.  It yields an array in one step without the need to manually run `explode()`.

Reply:
fixed